ui/app/serializers/capabilities.js

```diff
@@ -15,7 +15,12 @@ export default ApplicationSerializer.extend({
 
     if (requestType === 'query') {
       // each key on the response is a path with an array of capabilities as its value
-      response = Object.keys(payload.data).map((path) => ({ capabilities: payload.data[path], path }));
+      response = Object.keys(payload.data).map((fullPath) => {
+        // we use pathMap to normalize a namespace-prefixed path back to the relative path
+        // this is okay because we clear capabilities when moving between namespaces
+        const path = payload.pathMap ? payload.pathMap[fullPath] : fullPath;
+        return { capabilities: payload.data[fullPath], path };
+      });
     } else {
       response = { ...payload.data, path: payload.path };
```

Comment:
the other requestTypes we use (`findRecord` and `queryRecord`) are queried by the ID, which does not include the namespace. The payload.path is set to that ID on line 14, if it exists, ensuring that the id of the capability does not use the namespace-prefixed path even though the payload does include a namespace prefix on the path.